source/ui.js

```diff
@@ -98,7 +98,7 @@ module.exports = async (options, pkg) => {
 			type: 'list',
 			name: 'tag',
 			message: 'How should this pre-release version be tagged in npm?',
-			when: answers => !pkg.private && version(answers.version).isPrerelease() && !options.tag,
+			when: answers => !pkg.private && version.isPrereleaseOrIncrement(answers.version) && !options.tag,
```

Comment:
Huh?

Reply:
What's the issue here? I can't see anything out of the ordinary.

Reply:
Nvm. I missed https://github.com/sindresorhus/np/pull/368#issuecomment-473349716.